src/components/tl/ucp/tl_ucp_context.c

```diff
@@ -166,10 +166,8 @@ UCC_CLASS_INIT_FUNC(ucc_tl_ucp_context_t,
 
     ucp_params.field_mask =
         UCP_PARAM_FIELD_FEATURES | UCP_PARAM_FIELD_TAG_SENDER_MASK | UCP_PARAM_FIELD_NAME;
-    ucp_params.features = UCP_FEATURE_TAG | UCP_FEATURE_AM;
-    if (params->params.mask & UCC_CONTEXT_PARAM_FIELD_MEM_PARAMS) {
```

Comment:
why removing this "if"?

Reply:
This was removed to enable onesided collectives when using this API. Leaving the if statement would require users to pass memory map parameters to the context during context creation and then map memory using ucc_mem_map afterward. I think that would break the usability for programming models like MPI